mvg/analysis_classes.py

```diff
@@ -133,7 +175,7 @@ def feature(self):
         feature: str
         """
 
-        return self.raw_results["feature"]
+        return self._raw_results["feature"]
 
     def request_params(self):
```

Comment:
Both this method and `status` return `self._inputs`, is this intentional

Reply:
No, duplicate function